docs/source/guide/examples/prime_counting.py

```diff
@@ -83,7 +83,7 @@ def _create_cancel_button(self, parent):
             "Cancel", QtGui.QDialogButtonBox.RejectRole
         )
         self._cancel_button.setDefault(True)
-        buttons.rejected.connect(self.cancel)
+        buttons.rejected.connect(self.cancel, type=QtCore.Qt.QueuedConnection)
```

Comment:
Accidental unrelated change. Reverting.

```suggestion
        buttons.rejected.connect(self.cancel)
```